### PR TITLE
Adds additional unit for degrees, with option to include it in csv

### LINKED ISSUE #39: Units of angle in converted csv file
While converting *.BFL file to *.csv in the headers of csv there are units of barometer (cm), rotational velocity in ex. (deg/s) and acceleration (m/s/s) if the units were selected by command line switches. But there in no command line switch to select units of attitude provided when --simulate-imu is activated.

The orientation angles are not raw values,, are in (deg). To have consistent file, the units of angle should be added to roll, pitch and yaw in headers

It is now:
accSmooth[0] (m/s/s)   accSmooth[1] (m/s/s)   accSmooth[2] (m/s/s)   roll   pitch   heading

It should be:
accSmooth[0] (m/s/s)   accSmooth[1] (m/s/s)   accSmooth[2] (m/s/s)   roll(deg)   pitch(deg)   heading(deg)

To see it, it is necessary to use --simulate-imu option

### PR DESCRIPTION
**Pull request by dellch**
Fixes #39 

Adds additional unit for degrees, _"deg"_, with option to include it in csv for roll/pitch/heading.  

This additional unit will not be included in the headers unless the (newly created) flag `--include-imu-degrees` is included in the command line.  It also requires `--simulate-imu` to actually include those columns in the .csv output

### CODE REVIEW

**Comment by dellch** [comment]
This fixes #39 in a way that hopefully won't break any existing integrations, because it requires an additional CLI flag.  

Tried to follow standards - please let me know if there are any issues, or if the cli flag should be renamed.  

Also, been a _looong_ time since C, so excuse any glaring mistakes.

**Comment by dellch** [comment]
Also, nobody would want the unit to be in radians, right?

File: src/blackbox_decode.c [hunk 1]
```diff
@@ -488,11 +491,11 @@ void outputGPSFrame(flightLog_t *log, int64_t *frame)
         gpsFrameTime = lastFrameTime;
     }
 
-	bool haveRequiredFields = log->gpsFieldIndexes.GPS_coord[0] != -1 && log->gpsFieldIndexes.GPS_coord[1] != -1 && log->gpsFieldIndexes.GPS_altitude != -1;
-	bool haveRequiredPrecision = log->gpsFieldIndexes.GPS_numSat == -1 || frame[log->gpsFieldIndexes.GPS_numSat] >= MIN_GPS_SATELLITES;
+    bool haveRequiredFields = log->gpsFieldIndexes.GPS_coord[0] != -1 && log->gpsFieldIndexes.GPS_coord[1] != -1 && log->gpsFieldIndexes.GPS_altitude != -1;
```
Comment (dellch) [inline src/blackbox_decode.c:494]: these spacing changes were accidental, probably by my IDE - I'll work on reverting them

Comment (dellch) [reply src/blackbox_decode.c:494]: actually, it looks like the spacing is correct now - I think my IDE automatically converted all tabs to spaces.  Please let me know if I should fix this.  I didn't want to clutter the PR

File: src/blackbox_decode.c [hunk 2]
```diff
@@ -884,7 +887,11 @@ void writeMainCSVHeader(flightLog_t *log)
     }
 
     if (options.simulateIMU) {
-        fprintf(csvFile, ", roll, pitch, heading");
+        if (options.includeIMUDegrees){
+            fprintf(csvFile, ", roll (%s), pitch (%s), heading (%s)", UNIT_NAME[options.unitDegrees], UNIT_NAME[options.unitDegrees], UNIT_NAME[options.unitDegrees]);
+        }else{
```
Comment (haslinghuis) [inline src/blackbox_decode.c:892]: :space_invader: 
```suggestion
        } else {
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Spacing seems ok now, thanks, one styling change left.

File: src/blackbox_decode.c (hunk 2 above)
Comment (dellch) [reply src/blackbox_decode.c]: Thanks for catching that

**Review by McGiverGim — APPROVED** [review]
To me the code is ok. I don't know if it will be a very useful option, but is optional.

**Review by haslinghuis — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Comment by mikeller** [comment]
Manually tested the build - it's working.

**Pull request merged**